new_arrivals_chi/app/main.py

```diff
@@ -361,11 +362,37 @@ def health_search():
     Returns:
         Renders the health search page.
     """
+    # Implementation with demonstrative data
+    conn = sqlite3.connect("instance/test_fake_data.db")
+
+    cursor = conn.cursor()
+
+    cursor.execute(
+        "SELECT s.service, l.neighborhood, o.name, "
+        + "strftime('%I:%M %p', h.opening_time) "
+        "|| ' - ' || "
+        "strftime('%I:%M %p', h.closing_time) "
+        "AS opening_closing_time "
+        "FROM organizations o "
+        "JOIN hours h ON o.hours_id = h.id "
+        "JOIN locations l ON o.location_id = l.id "
+        "JOIN organizations_services os ON o.id = os.organization_id "
+        "JOIN services s ON os.service_id = s.id"
+    )
+
+    services_info = cursor.fetchall()
+
+    conn.close()
+
```

Comment:
We can make an issue to do this with the real db, where I think we will not use string-like SQL queries